<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BankAccount</class>
 <widget class="QWidget" name="BankAccount">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>441</width>
     <height>391</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="accountsTab">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QLabel" name="lblBalance0">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>80</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblBalance1">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>220</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblType0">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Checking:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblType1">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>200</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Savings: </string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="transferTab">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
    <widget class="QLabel" name="lblTransferFrom">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Transfer From: </string>
     </property>
    </widget>
    <widget class="QLabel" name="lblTransferTo">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>180</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Transfer To: </string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbFrom">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>211</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="cbTo">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>190</y>
       <width>211</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="submitTransfer">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>320</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Transfer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cancelTransfer">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>320</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="txtAmt">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>250</y>
       <width>104</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblAmount">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>250</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Amount:</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
